wca-regulations.md

```diff
@@ -90,6 +90,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
     - 2j1) If a competitor is disqualified from an event for any reason, they are not eligible for any more attempts in the event.
         - 2j1a) The results of all remaining attempts in the event are recorded as DNF.
     - 2j2) If a competitor is disqualified during the course of an event, their earlier results remain valid. Exception: cheating or defrauding (see [Regulation 2k2a](regulations:regulation:2k2a)).
+    - 2j3) If a competitor does not fulfill the event's requirements (e.g. know how to solve the puzzle). A competitor must not compete with the expectation of a DNF result or an intentionally poor result.
```

Comment:
I think this might be better.

If a competitor does not fulfill the event's requirements (e.g. know how to solve the puzzle)**, the** competitor must not compete with the expectation of a DNF result or an intentionally poor result.

Reply:
The original wording contains a fragmebted sentence. The new wording does not carry the correct meaning (the reg would not ban knowing how to solve the puzzle, but simply wasting time)

Reply:
@hanwu85 all 2j uses `a` instead of `the`. So we can change all 2j or keep as how it is.

> The original wording contains a fragmented sentence. The suggested wording does not carry the correct meaning (the reg would not ban knowing how to solve the puzzle, but simply wasting time)
Yes, I will correct it!